sotodlib/toast/workflows/proc_flagging.py

```diff
@@ -72,13 +72,15 @@ def setup_flag_noise_outliers(operators):
         toast.ops.FitNoiseModel(
             name="noise_cut_fit",
             out_model="noise_cut_fit",
+            enabled=False,
```

Comment:
Same comment as above, let's leave the fitting / flagging enabled and just toggle the estimation operator.